wgsmetrics.wdl

```diff
@@ -0,0 +1,56 @@
+version 1.0
+
+workflow wgsMetrics {
+input {
+    File    inputBam
+    String? refFasta = "$HG19_ROOT/hg19_random.fa"
+    String? outputFileNamePrefix = ""
+    String? stringencyFilter = "LENIENT"
+    String? modules = "java/8 picard/2.19.2 hg19/p13"
+    Int?    coverageCap = 500
+}
+
+call collectWGSmetrics{ input: inputBam = inputBam, refFasta = refFasta, filter = stringencyFilter, coverageCap = coverageCap, outputPrefix = outputFileNamePrefix, modules = modules }
+
+output {
+  File outputWGSMetrics  = collectWGSmetrics.outputWGSMetrics
+}
+
+}
+
+# ==========================================
+#  TASK 1 of 1: collect WGS metric
+# ==========================================
+task collectWGSmetrics {
+input { 
+        File   inputBam
+        String? refFasta
+        String? metricTag  = "WGS"
+        String? filter     = "LENIENT"
+        String? outputPrefix = ""
```

Comment:
Provide default `outputPrefix` otherwise the output file would be named `.WGS`

Reply:
fixed, see line 9 of wgsmetrics.wdl